packages/client/src/l2.write.ts

```diff
@@ -144,4 +144,41 @@ const _ = (async () => {
       console.error('error setting text: ', { err })
     }
   }
+
+  // SET ADDRESS
+  try {
+    await client.simulateContract({
+      functionName: 'setAddr',
+      abi: l1Abi,
+      args: [
+        toHex(packetToBytes(publicAddress)),
+        '0x04270c4366010A52192bC8D3E29d9f0E21bBe969',
+      ],
+      address: resolverAddr,
+    })
+  } catch (err) {
+    const data = getRevertErrorData(err)
+    if (data?.errorName === 'StorageHandledByL2') {
+      const [chainId, contractAddress] = data.args as [bigint, `0x${string}`]
+
+      await handleL2Storage({
+        chainId,
+        l2Url: providerL2,
+        args: {
+          functionName: 'setAddr',
+          abi: l2Abi,
+          args: [
+            namehash(publicAddress),
+            '0x04270c4366010A52192bC8D3E29d9f0E21bBe969',
+          ],
+          address: contractAddress,
+          account: signer,
+        },
+      })
+    } else if (data) {
+      console.error('error setting addr: ', data.errorName)
```

Comment:
```suggestion
      console.error('error setting addr. data.errorName is: ', data.errorName)
```

Reply:
I suggest it because we had to prints with the same console.

Reply:
what do you mean?